content/docs/security/local-ratelimit.md

```diff
@@ -261,8 +255,154 @@ Instead of applying local rate limiting to a particular route, you can also appl
    local_rate_limited      
    ```
 
-5. Remove the resources that you created in this guide. 
+## Disable rate limiting for a route {#disable-route}
+
+Sometimes, you might want to disable rate limiting for a route. For example, you might have system critical routes that should be accessible even under high traffic conditions, such as a health check or admin endpoints. You can exclude a route from rate limiting by setting the `rateLimit.local` to `{}` in the TrafficPolicy. 
+
+1. Create a Gateway-level TrafficPolicy to enforce local rate limiting on all routes. For more information, refer to the [Gateway configuration](#gateway).
+
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.kgateway.dev/v1alpha1
+   kind: TrafficPolicy
+   metadata:
+     name: local-ratelimit
+     namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+   spec:
+     targetRefs: 
+     - group: gateway.networking.k8s.io
+       kind: Gateway
+       name: http
+     rateLimit:
+       local:
+         tokenBucket:
+           maxTokens: 1
+           tokensPerFill: 1
+           fillInterval: 100s
+   EOF
+   ```
+
+2. Create an HTTPRoute for the route that you want to exclude from rate limiting, such as `/anything` on the `httpbin` app.
+
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.networking.k8s.io/v1
+   kind: HTTPRoute
+   metadata:
+     name: httpbin-anything
+     namespace: httpbin
+   spec:
+     parentRefs:
+     - name: http
+       namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+     hostnames:
+     - www.example.com
+     rules:
+     - matches:
+       - path:
+           type: PathPrefix
+           value: /anything
+       backendRefs:
+       - name: httpbin
+         port: 8000
+   EOF
+   ```
+
+3. Send two requests to verify that the route is rate limited due to the Gateway-level TrafficPolicy that allows only 1 request per 100 seconds.
+
+   {{< tabs items="LoadBalancer IP address or hostname,Port-forward for local testing" >}}
+   {{% tab  %}}
+   ```sh
+   curl -vi http://$INGRESS_GW_ADDRESS:8080/anything -H "host: www.example.com:8080" && curl -vi http://$INGRESS_GW_ADDRESS:8080/anything -H "host: www.example.com:8080"
```

Comment:
you could do 

`for i in {1..2}; do curl -vi http://$INGRESS_GW_ADDRESS:8080/anything -H "host: www.example.com:8080" && curl -vi http://$INGRESS_GW_ADDRESS:8080/anything -H "host: www.example.com:8080"; done`